tests/cases/fourslash/overloadOnConstCallSignature.ts

```diff
@@ -11,8 +11,8 @@
 
 goTo.marker('1');
 verify.signatureHelpCountIs(4);
-verify.currentSignatureHelpIs('foo(name: string): string');
+verify.currentSignatureHelpIs('foo(name: \'order\'): string');

```

Comment:
Yeah, in general the philosophy for signature help is to give the user a lens into the ordering of the signatures. In that respect I think seeing the specialized first is better. So I am good with the way you have it.